src/components/Animal/Animal.js

```diff
@@ -4,7 +4,7 @@ export default function Animal(props) {
   return (
     <>
       <div className="animal" style={{ top: props.top, left: props.left }}>
-        {props.name} {props.type} {props.says}
+        {props.name} {props.says}
```

Comment:
It is best practice to use a <p> tag or a <span> here to wrap the text, instead of having text directly in your <div> element.

Reply:
Awesome! Thanks for letting me know; that's super helpful.